File: member/mall-api/src/main/java/com/laf/mall/api/dto/RaffleLog.java
package com.laf.mall.api.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.Date;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class RaffleLog {
    private int trlog_id;

    private int raffle_id;

    private int member_id;

    private String member_name;

    private String member_mobile;

    private int trr_id;

    private Date action_time;

    public long getAction_time() {
        return action_time == null ? 0L : action_time.getTime();
    }
}

File: member/manager-app/src/main/java/com/laf/manager/dto/RaffleLog.kt
package com.laf.manager.dto

import java.util.*

data class RaffleLog(
        //i.title,l.member_name,l.member_mobile,i.required_points,l.trr_id,l.action_time,r.reward_name,r.reward_type
//        var r_id: Int,
        var title: String,
        var member_name: String,
        var member_mobile: String,
        //        var win: Boolean,
        var action_time: Date,
        var required_points: Int,
        var trr_id: Int,
        var reward_name: String?,
        var reward_type: Int

) {
}
